src/Stratis.Bitcoin.IntegrationTests/Wallet/SendingToAndFromManyAddressesSpecification_Steps.cs

```diff
@@ -93,18 +93,17 @@ private void Mine100Coins(CoreNode node)
 
         private void node2_receives_the_funds()
         {
+            // Need to mine a block prior to checking for spendable transactions in node2's wallet.
+            this.sharedSteps.MineBlocks(1, this.nodes[NodeOne], WalletAccountName, WalletName, WalletPassword);
```

Comment:
Why does the block need to be mined?
The transaction will appear in node2 and be spendable even with 0 confirmations.